js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -99,6 +100,20 @@ export class MiniAppBridge {
       );
     });
   }
+
+  /**
+   * Associating showInterstitialAd function to MiniAppBridge object
+   */
+  showInterstitialAd() {
```

Comment:
Need to be updated to align with js sdk `MiniAppBridge.showInterstitialAd(id)`.

Reply:
Yes we need to update this along with other updates!
Sorry! please re-consider review
Thanks!